services/pull/merge.go

```diff
@@ -53,6 +53,9 @@ func getMergeMessage(ctx context.Context, baseGitRepo *git.Repository, pr *issue
 		issueReference = "!"
 	}
 
+	reviewedOnTrailer := fmt.Sprintf("Reviewed-on: %s/%s", setting.AppURL, pr.Issue.Link())
+	reviewedByTrailer := pr.GetApprovers(ctx)
```

Comment:
Note/Question: Should I also migrate the `GetApprovers` function into `merge.go` (since it is only used here and nowhere else)?

Reply:
Technically yes.
However, I don't see this as an obligation for this PR.
If you want to, go ahead, but I'm not forcing you.

Reply:
Actually, I noticed that `GetApprovers` is in `models/issues/pull.go` and is a member function. Moving it to `merge.go` would thus force making it a free function. I think it doesn't make sense to move it then.